cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -423,8 +423,7 @@ func buildMeshRootCertificate(presetMeshRootCertificateConfigMap *corev1.ConfigM
 		},
 		Spec: presetMeshRootCertificateSpec,
 		Status: configv1alpha2.MeshRootCertificateStatus{
-			State:         constants.MRCStateComplete,
-			RotationStage: constants.MRCStageIssuing,
+			ProviderState: constants.MRCStateActive,
```

Comment:
I'm still not sure ProviderState is a great term. What is the provider? I believe it is one of tresor, vault or cert manager.

In that sense, this state does not describe the provider, but instead describes the state of the meshrootcertificate.

How about we just call it `State`?

cc @jaellio

Reply:
I'm indifferent on the naming - whatever y'all decide is best :)

Reply:
I am fine with using `State`! I liked provider state because it was describing how the provider was being used. Since other fields in the MRC can be updated without having to create a new MRC I liked making the state specific to the provider.

Reply:
I will approve once `ProviderState` is updated to `State`. Thanks @schristoff!